### PR TITLE
Release/gmail login

### PR DESCRIPTION
**Pull request by ricardobrg**
# Issues Addressed
#235
#236
#237
#252
#154
#156
#157
#159

### Known issues
The email scan takes too long to run as a regular background task. We are reviewing the architecture to address this issue and run the email and retailer scans as OS background tasks in Android and iOS, taking advantage of the OS-specific background APIs.

### CODE REVIEW

File: src/receipt-capture-plugin.ts [hunk 1]
```diff
@@ -6,18 +6,18 @@
...
+  username: string;
+  password: string;
+  source: string;
 }
 
-export interface ReqInitialize{
+export interface ReqInitialize {
   licenseKey: string;
   productKey: string;
+  googleId: string | undefined;
```
Comment (mike-audi) [inline src/receipt-capture-plugin.ts:20]: @JesseMonteiro & @ricardobrg can you explain what this is and why it's in the Initialization obj?

Comment (ricardobrg) [reply src/receipt-capture-plugin.ts:20]: Looks like it is missing the typescript documentation and readme. I need to add those.

This is the Google Client Id for Google OAuth. If it is not provided IMAP will be used for Gmail.

**Review by mike-audi — CHANGES_REQUESTED** [review]
per @ricardobrg's comment: Looks like it is missing the typescript documentation and readme. I need to add those.

File: src/receipt-capture-plugin.ts (hunk 1 above)
Comment (ricardobrg) [reply src/receipt-capture-plugin.ts]: see [4b86e42](https://github.com/tiki/capture-receipt-capacitor/pull/265/commits/4b86e4281595c672aa4267e3b8e0fcf120872ffc)

**Review by mike-audi — APPROVED** [review]

**Pull request merged**